### PR TITLE
Update match.md

### PR DESCRIPTION
**Pull request by svitlo**
Corrected the name of the metric used in the field  `fuzziness`. It is called the Damerau–Levenshtein distance because the field  `fuzzy_transpositions` is equal to `true` by default.

### Description
The change corrected the name of the metric used in the field `fuzziness` by default. Similar correction is requested on the [PR 7842](https://github.com/opensearch-project/documentation-website/pull/7842).

### Issues Resolved
No issues resolved

### Version
All

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by svitlo** [comment]
@hdhalter, I am ready for your doc review.

**Comment by svitlo** [comment]
By the way I corrected the description of the field `fuzziness` as well.

File: _query-dsl/full-text/match.md [hunk 1]
```diff
@@ -454,12 +454,12 @@ Parameter | Data type | Description
...
 `enable_position_increments` | Boolean | When `true`, resulting queries are aware of position increments. This setting is useful when the removal of stop words leaves an unwanted "gap" between terms. Default is `true`.
-`fuzziness` | String | The number of character edits (insert, delete, substitute) that it takes to change one word to another when determining whether a term matched a value. For example, the distance between `wined` and `wind` is 1. Valid values are non-negative integers or `AUTO`. The default, `AUTO`, chooses a value based on the length of each term and is a good choice for most use cases.
+`fuzziness` | String | The number of character edits (insert, delete, substitute, transpose) that it takes to change one word to another when determining whether a term matched a value. For example, the distance between `wined` and `wind` is 1. Valid values are non-negative integers or `AUTO`. The default, `AUTO`, chooses a value based on the length of each term and is a good choice for most use cases.
```
Comment (kolchfa-aws) [inline _query-dsl/full-text/match.md:457]:
```suggestion
`fuzziness` | String | The number of character edits (insertions, deletions, substitutions, or transpositions) that it takes to change one word to another when determining whether a term matched a value. For example, the distance between `wined` and `wind` is 1. Valid values are non-negative integers or `AUTO`. The default, `AUTO`, chooses a value based on the length of each term and is a good choice for most use cases.
```

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @svitlo! One suggestion. Please fix DCO so we can merge this PR. There are also a couple of other places that we mention Levenshtein distance (/_field-types/supported-field-types/completion.md, /_query-dsl/term/index.md) if you want to take those on as part of this PR 😄

**Comment by svitlo** [comment]
> Please fix DCO so we can merge this PR

@kolchfa-aws, Fixed.

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**